backend/app/resources/ResourceEndpoint.py

```diff
@@ -104,7 +104,10 @@ class EducationResourceListEndpoint(flask_restful.Resource):
     resourcesSchema = ResourceSchema(many=True)
 
     def get(self):
-        resources = db.session.query(Resource).filter_by(is_uva_education_content=True).all()
+        resources = db.session.query(Resource)\
+            .filter_by(is_uva_education_content=True, is_draft=False)\
```

Comment:
We should standardize on filter or filter_by.  I like filter better for what it is worth, it's a little more verbose, and quicker to catch errors.